components/scream/cime_config/namelist_defaults_scream.xml

```diff
@@ -238,7 +247,8 @@ tweaking their $case/namelist_scream.xml file.
     ${DIN_LOC_ROOT}/atm/scream/tables/vn_table_vals.dat8,
     ${DIN_LOC_ROOT}/atm/scream/tables/vm_table_vals.dat8,
     ${DIN_LOC_ROOT}/atm/scream/init/spa_file_unified_and_complete_ne30_20220428.nc,
-    ${DIN_LOC_ROOT}/atm/scream/init/map_ne30_to_ne4_mono_20220502.nc
+    ${DIN_LOC_ROOT}/atm/scream/init/map_ne30_to_ne4_mono_20220502.nc,
+    ${DIN_LOC_ROOT}/atm/scream/init/map_ne30np4_to_ne120np4_mono_20220502.nc
```

Comment:
Do we want to include IC files in this list of things to download? Note that trying to download all files for all resolutions won't be a workable long-term strategy (but is probably ok for v1).

Reply:
Yes, that's a good point. We should use cime vars to download just _one_ IC file, depending on the grid resolution.

Reply:
Well, right now we're not downloading any _IC_ file, but we are downloading all _remap_ files. I'm inclined to have folks manually download both IC files and mapping files for SCREAMv1 since we're all power users and aren't running on that many machines.